### PR TITLE
prooph cs fixer test image.

### PR DESCRIPTION
**Pull request by bweston92**
image for prooph cs fixer #8

### CODE REVIEW

**Comment by prolic** [comment]
LGTM. @sandrokeil can you check this? You're the pro here :)

**Comment by bweston92** [comment]
I would drop the composer baggage as it could save a few megabytes using
the new pipeline stuff in Dockerfiles but not sure on whether the thing
that builds these images will be up to date.

File: php-cs-test/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Introduction
+
+The aim of this container image is to eliminate the need to `php-cs-fixer` in the composer requirements for each project we do. This image can be used on any environment that has Docker installed including a CI pipeline.
+
+# Usage
+
+Mount code to `/sut` on the container and we will test again prooph's rules.
```
Comment (codeliner) [inline php-cs-test/README.md]: Has `sut` a special meaning? In other containers (f.e. prooph/php-cli) we mount $(pwd) to `/app`
Would it be ok to do the same here?

**Comment by bweston92** [comment]
I just chose SUT for "subject under test" can do anything as long as
entrypoint is updated.

**Comment by codeliner** [comment]
ah ok, I like it! But I think it is better to keep it consistent with the other images and mount it to `app`

**Comment by bweston92** [comment]
That's fine, will update.

File: php-cs-test/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,10 @@
+FROM composer
+
```
Comment (sandrokeil) [inline php-cs-test/Dockerfile]: We should use `prooph/composer:7.1` here

File: php-cs-test/README.md (hunk 1 above)
Comment (sandrokeil) [reply php-cs-test/README.md]: @bweston92  Yep, please use `/app`.

File: php-cs-test/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+RUN composer global require prooph/php-cs-fixer-config
+
+COPY rules.php /rules.php
+
+VOLUME /sut
+WORKDIR /sut
+
+ENTRYPOINT ["/composer/vendor/bin/php-cs-fixer", "fix", "--dry-run", "--diff", "--config=/rules.php"]
```
Comment (sandrokeil) [inline php-cs-test/Dockerfile]: @bweston92 What do you think about these lines, so the CMD arguments can be overridden?

```
CMD ["fix", "--dry-run", "--diff"]
ENTRYPOINT ["/composer/vendor/bin/php-cs-fixer", "--config=/rules.php"]
```

The composer path is another path with `prooph/composer`.

File: php-cs-test/Dockerfile (hunk 2 above)
Comment (bweston92) [reply php-cs-test/Dockerfile]: I will look on the image sizes, but I'd imagine a bare composer base image would be less in size.

File: php-cs-test/Dockerfile (hunk 3 above)
Comment (bweston92) [reply php-cs-test/Dockerfile]: I can do, I was more thinking this image is the test and therefore wouldn't be run any different. Will update PR soon.

File: php-cs-test/README.md (hunk 1 above)
Comment (bweston92) [reply php-cs-test/README.md]: Done

File: php-cs-test/Dockerfile (hunk 2 above)
Comment (bweston92) [reply php-cs-test/Dockerfile]: I've gone ahead and used composer as the official image still as it only needs to collect the file in that step.

File: php-cs-test/Dockerfile (hunk 3 above)
Comment (bweston92) [reply php-cs-test/Dockerfile]: Done

**Comment by bweston92** [comment]
@sandrokeil can you take a look at it.

**Pull request merged**